src/app/page.tsx

```diff
@@ -5,18 +5,17 @@ import styles from "./page.module.css";
 import InputText from "~/ui/InputText";
 import BtnAdd from "~/ui/BtnAdd";
 
-
 export default function Home() {
   return (
-    <div>
+    <div className="todo-list">
       <h1>To Do List</h1>
-      <div>
+      <div className="todo-list-inputText">
         <InputText onChange={(text) => {
           console.log(text);
         }}/>
       </div>
-      <div>
-        <BtnAdd className="btn-class">Enviar</BtnAdd>
+      <div className="todo-list-btnAdd">
```

Comment:
this style makes no sense, as there's no definition for it, please remove it

Reply:
👍